ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -223,6 +223,14 @@ class MSFListCellStateImpl: NSObject, ObservableObject, Identifiable, ControlCon
         return children.count
     }
 
+    var trailingItem: MSFListAccessoryType {
```

Comment:
Do we need to have this stored in the state object?
What I had in mind is to calculate this on the rendering of the view body instead and store this in a let constant.